src/bun.js/webcore/response.zig

```diff
@@ -495,12 +501,15 @@ pub const Response = struct {
                     break :brk Body.extract(globalThis, arguments[0]);
                 },
                 else => {
+                    if (arguments[1].isUndefinedOrNull()) break :brk Body.extract(globalThis, arguments[0]);
                     switch (arguments[1].jsType()) {
                         .Object, .FinalObject, .DOMWrapper => |js_type| {
                             break :brk Body.extractWithInit(globalThis, arguments[0], arguments[1], js_type);
                         },
                         else => {
```

Comment:
I don't think this JSType() check is right


It can be any of the function JSType's too

It can also be a `Blob`.

I think it has to be `.isObject()` (which checks if JSType > .Objectg)

Reply:
👍